bindings/gumjs/gumdukvalue.c

```diff
@@ -701,6 +701,23 @@ _gum_duk_get_int64 (duk_context * ctx,
   }
 }
 
+static gboolean
+gum_duk_is_int64 (duk_context *ctx,
+                  duk_idx_t index,
+                  GumDukCore *core)
```

Comment:
This one should be moved right above ` _gum_duk_get_int64()`, so they're ordered: `is`, `get`, `parse`.